site/pages/about.md

```diff
@@ -18,17 +18,17 @@ If you're planning to use TBIL in your classroom, you can get a link to join our
 
 ### Publications
 
-- Lewis, D., Clontz, S., and Estis, J. 2021. Team-Based Inquiry Learning. PRIMUS 31 (2), 223-238. <https://doi.org/10.1080/10511970.2019.1666440>
+- Lewis, D., Clontz, S., and Estis, J. (2021). Team-Based Inquiry Learning. _PRIMUS_ 31 (2), 223-238. <https://doi.org/10.1080/10511970.2019.1666440>
 
     - Team-Based Learning (TBL) is a cooperative learning strategy blending elements of flipped learning, inquiry-based learning, and problem-based learning. Although used quite frequently in other disciplines, use of this strategy in mathematics has been limited. In this article, we describe how TBL can be implemented in math courses with adherence to essential elements of TBL and introduce modifications specific to mathematics instruction. In particular, we introduce a particular style of TBL, which we term Team-Based Inquiry Learning, that satisfies the defining pillars of inquiry-based learning.
 
 
-- Lewis, D. and Estis, J. 2020. Improving Mathematics Content Mastery and Enhancing Flexible Problem Solving through Team-Based Inquiry Learning. Teaching & Learning Inquiry 8 (2), 165-183. <https://doi.org/10.20343/teachlearninqu.8.2.11>
+- Lewis, D. and Estis, J. (2020). Improving Mathematics Content Mastery and Enhancing Flexible Problem Solving through Team-Based Inquiry Learning. _Teaching & Learning Inquiry_ 8 (2), 165-183. <https://doi.org/10.20343/teachlearninqu.8.2.11>

```

Comment:
This link isn't working for me today---possibly a ucalgary issue?

Reply:
Yeah, I think just leave this one alone and it's really on the journal to fix their website.